lms/static/js/student_account/views/LoginView.js

```diff
@@ -86,6 +93,10 @@
 
             resetEmail: function() {
                 this.element.hide( this.$errors );
+                this.resetMessage = this.$resetSuccess.find('.message-copy');
+                if (this.resetMessage.find('p').length == 0){
+                    this.resetMessage.append("<p>" + this.message + "</p>");
```

Comment:
You can see the failure here:

https://build.testeng.edx.org/job/edx-platform-quality-pr/22206/artifact/edx-platform/reports/safecommit/safecommit.report/*view*/

Note that all the other failures are unrelated issues in the files you touched, so I think this is the only one that you should fix.